HWDataset.py

```diff
@@ -0,0 +1,90 @@
+from torch.utils.data import Dataset
+from torch.utils.data import DataLoader
+from pathlib import Path
+import math
+import numpy as np
+import plot_stroke
+import torch
+
+def collate_fn(samples):
+    batch_size=len(samples)
+    strokes = [s for s, text in samples]
```

Comment:
What do you think of this?
`strokes, texts = zip(*samples)`

Reply:
Nice, thanks.